hardware/products/ucm/intro.md

```diff
@@ -1,3 +1,249 @@
-# UCM Products Introduction
+# UCM Introduction
 
-Coming soon.
\ No newline at end of file
+What is UCM? UCM is a Universal Compute Module based on the 200-pin SO-DIMM, Small Outline Dual In-line Memory Module. To make these modules mostly interchangeable, GHI Electronics has made a standard pin format for these 200-pins.
+
+The standard consists of several widely used peripherals found on many microcontrollers such as UART, SPI, I2C, PWM, etc.
+
+The peripherals found on each module may vary.
+# UCM Standard
+## UCM Standard - Available Peripherals
+| Peripheral											| Up to Max	|
```

Comment:
Unify the spacing between the columns in this table and the others.

Reply:
Not sure what you mean by this.

Reply:
Take a look at the raw file, https://raw.githubusercontent.com/ghi-electronics/Docs/0d1dd040f2c4a6f76cec2a90c6b359d22b6e1f68/hardware/products/ucm/intro.md, the spacing is all inconsistent in the table so it's hard to read unless you use the rendered markdown view.

Reply:
With this, I used tab stops to keep things spaced correctly in the program that I used. Because tabs are different on different software, I did not consider that. I will have to use spaces to solve this.